frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -1,13 +1,8 @@
-import { Plugin } from '@console/plugin-sdk';
-import { getTopologyPlugin, TopologyConsumedExtensions } from './topology/topology-plugin';
-
 import '@console/internal/i18n.js';
 import './style.scss';
```

Comment:
These imports are side-effects, we should evaluate whether we still need them and if so, move their execution into appropriate components or functions.

Context: static plugin's entry module becomes part of auto-generated `@console/active-plugins` virtual module, processed by webpack during Console application build.